README.md

```diff
@@ -1,38 +1,51 @@
-# Morpho Token
+# Morpho token
 
-This repository contains the Morpho protocol’s ERC20 token.
+This repository contains the Morpho protocol's ERC20 token.
 It is designed to be upgradable and support onchain delegation.
 Additionally, it ships a wrapper contract to simplify the migration of assets from the previous token contract to the new Morpho token contract.
 
-## Upgradability
+## Features
 
-The Morpho token complies with the [EIP-1967](https://eips.ethereum.org/EIPS/eip-1967) to support upgradeability.
+### Upgradeability
 
-## Delegation
+The Morpho token complies with [EIP-1967](https://eips.ethereum.org/EIPS/eip-1967) to support upgradeability.
+
+### Delegation
 
 The Morpho token supports onchain voting and voting power delegation.
 
+### Role-based permission
+
+The new Morpho token does not have role-based permission of functions.
+
+### Burning tokens
+
+In the legacy Morpho token, burning tokens was made possible by transferring them to the zero address.
```

Comment:
the legacy token also had a [burn function](https://github.com/morpho-org/semitransferable-token/blob/dce1fe378e14ecf2886dcc6caf6007542290e4be/src/Token.sol#L46)

Reply:
Oh right, my bad. This is fixed now